File: Stories.API/Stories.API.Data/Models/Story.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Stories.API.Data.Models
{
    public class Story
    {
         public Story(string title, string description, string departament)
        {
            Title = title;
            Description = description;
            Departament = departament;
        }

        public int Id { get; private set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string Departament { get; set; }
        public ICollection<Vote> Votes { get; } = new List<Vote>();

    }
}

File: Stories.Web/src/app/models/Story.ts
import { Vote } from './Vote';

export class Story {
  constructor(
    id: number,
    title: string,
    description: string,
    departament: string,
    votes: Array<Vote>
  ) {
    this.id = id;
    (this.title = title), (this.description = description);
    this.departament = departament;
    this.votes = votes;
  }

  id: number;
  title: string;
  description: string;
  departament: string;
  votes: Array<Vote>;
}
